Task3/UserTaskController.cs

```diff
@@ -13,30 +14,18 @@ public UserTaskController(UserTaskService taskService)
 
         public bool AddTaskForUser(int userId, string description, IResponseModel model)
         {
-            string message = GetMessageForModel(userId, description);
-            if (message != null)
+            try
             {
-                model.AddAttribute("action_result", message);
-                return false;
-            }
-
-            return true;
-        }
-
-        private string GetMessageForModel(int userId, string description)
-        {
-            var task = new UserTask(description);
-            int result = _taskService.AddTaskForUser(userId, task);
-            if (result == -1)
-                return "Invalid userId";
+                var task = new UserTask(description);
+                _taskService.AddTaskForUser(userId, task);
 
-            if (result == -2)
-                return "User not found";
-
-            if (result == -3)
-                return "The task already exists";
-
-            return null;
-        }
+                return true;
+            }
+            catch (Exception ex)
```

Comment:
It s possible to get another Exception type in this block, if this error won't be expected by you , RuntimeException fo example, the behavior of app will change. As it possible to get unexpected action_result message.


Pls catch only exception types you want to handle